cloud-azure/src/test/java/com/sequenceiq/cloudbreak/cloud/azure/AzureEncryptionResourcesTest.java

```diff
@@ -298,6 +298,7 @@ public void testCreateDiskEncryptionSetShouldReturnExistingDiskEncryptionSetWith
         when(azureClient.keyVaultExists("dummyResourceGroup", "dummyVaultName")).thenReturn(Boolean.TRUE);
         when(azureClient.checkKeyVaultAccessPolicyForServicePrincipal("dummyResourceGroup", "dummyVaultName", DES_PRINCIPAL_ID))
                 .thenReturn(true);
+        //when(azureClient.isRoleBasedAccessControlEnabled("dummyResourceGroup", "dummyVaultName")).thenReturn(false);
```

Comment:
Yes, this UT shall be amended to cover both kinds of KeyVault access, during both creation and deletion.